### DISCUSSION TITLE
EmotiBit, perhaps an option for a watchless future?

### DISCUSSION THREAD

**Discussion by lundstrj** [OpenSeizureDetector/Android_Pebble_SD#159] [04_Devices]
Oxana (that would be the wife https://github.com/merenlin) got [this](https://www.emotibit.com/) for their IoT-lab and she says it was really nice to work with (they also tried a plethora of other COTS wearables, such as Fitbit, BangleJS, Pinetime etc).

The built in sensors were of high quality and they could get high resolution accelerometer data out of it. The HRM does suffer a bit when the subject is moving but all optical HRMs have that problem. Granted, the EmotiBit probably does more than what we need for OSD but the entire device is open source and a lite version could in theory be designed with features and sensors stripped away.

I personally like that it has a thermometer which supposedly is pretty accurate when worn (reporting her coworker at 36.5-37 degrees Centigrade). It would be very useful to get an early warning on a spiking fever so we can avoid the really high highs (which in the case of our son, has the unfortunate tendency to also trigger seizures).

The same company also develops, makes and sells an (among other things) [EEG machine](https://shop.openbci.com/collections/frontpage). Granted it isn't exactly free but I find it absolutely wild that a functional, supposedly pretty darn good, EEG can be bought by a consumer for non astronomical figures.

**Comment by pmithrandir** [comment]
Hi,

From a general point of view, I would say that any device should be investigated.
That being said, I have few remarks on this one: 
- it doesn't seem to be ready to go out with a disable kid playing outside not knowing the value of the device. 
- it's not cheap (50% more than pinetime)

Do you think it could be usable in a real context with real people living their life, or is it more a lab device ?

**Reply by lundstrj** [reply]
Agreed. This is a development board. We do not need all it has to offer. I do like that it seems to have higher quality sensors and I am tempted to look into what it would cost to make a simpler cheaper board with only HRM, accelerometer and thermometer and then limit connectivity to bluetooth and perhaps have a little multicolor LED and as few buttons as possible. Maybe see what kind of battery would be needed to get at least a couple of days worth of usage between charges. Stick it in a nice waterproof case. My hope is that is can be made smaller and lighter and that it might simplify the overall system.

Then again, all of this listed above, is no small feat to accomplish. Maybe just stick to the PineTime and polish that experience instead?

**Reply by jones139** [reply]
I did set out to design my own watch with a FreeRTOS based system running on it (https://github.com/OpenSeizureDetector/OSDWatch/tree/master)  I only really got to the breadboard state though (https://www.openseizuredetector.org.uk/?p=1115) and realised how big a job it was going to be to make something that came close to what Garmin etc. produce.
...Then I discovered PineTime and thought that it was so close to being what I would have built that I should use that.  (An O2 saturation sensor would be nice though, but even the Garmin ones are not very good at that).
